### PR TITLE
feat(model): adds properties, ownership, deprecated, institutional memory and tags as aspects for data platform instance entity

### PR DESCRIPTION
**Pull request by sgomezvillamor**
Adds properties, ownership, deprecated, institutional memory and tags as aspects for data platform instance entity.

We may also add the corresponding `EditableDataPlatformInstanceProperties`, this would require a UI where we can view and update a Data Platform Instance entity. Does exist such a page view?

I have explicitely skipped `GlossaryTerm` and `Domain` because those look more related to data than infrastructure
 
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by sgomezvillamor** [comment]
@shirshanka , any feedback here?

**Comment by sgomezvillamor** [comment]
@hsheth2 could you help me here? what else is needed to extend the model?

File: metadata-models/src/main/pegasus/com/linkedin/dataplatforminstance/DataPlatformInstanceProperties.pdl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  @Searchable = {
+    "fieldType": "TEXT",
+    "hasValuesFieldName": "hasDescription"
+  }
+  description: optional string
+
+  /**
+   * The abstracted URI, usually host and port. Uri should not include any environment specific properties.
+   */
+  uri: optional Uri
```
Comment (jjoyce0510) [inline metadata-models/src/main/pegasus/com/linkedin/dataplatforminstance/DataPlatformInstanceProperties.pdl:37]: Can we actually remove this? i mean - do we strictly need it?

**Review by jjoyce0510 — APPROVED** [review]
Overall LGTM. Just had one question then I think we can get this in!

**Comment by jjoyce0510** [comment]
We do not yet have an entity profile page in the UI for Data Platform Instance. That's something we'll have to add post this PR.

**Comment by sgomezvillamor** [comment]
Thanks @jjoyce0510

**Pull request merged**